internal/credentials/credentials.go

```diff
@@ -95,7 +105,7 @@ func NewCredentialsService(log logging.Logger) *credentialsService {
 // FileCredentialRecordFactory creates a Credential based on the presence of the
 // a file containing CONNECT_SERVER and CONNECT_API_KEY.
 
-type fileCredential struct {
+type deprcfileCredential struct {
```

Comment:
Marking the current `fileCredential` struct as deprecated, will be removed in the next PR and be replaced by `file.go#fileCredential`